lib/bolt/executor.rb

```diff
@@ -212,6 +216,7 @@ def run_task(targets, task, arguments, options = {}, &callback)
       log_action(description, targets) do
         notify = proc { |event| @notifier.notify(callback, event) if callback }
         options = { '_run_as' => run_as }.merge(options) if run_as
+        options = { '_load_config' => @load_config }.merge(options)
```

Comment:
This should be `options = options.merge { '_load_config' => @load_config }` to prevent the caller overriding the executor.